app/models/access_token.rb

```diff
@@ -0,0 +1,52 @@
+class AccessToken < ActiveRecord::Base
+  belongs_to :repository
+  attr_accessible :expiration, :token
+
+  scope :expired, ->() { where('expiration <= ?', Time.now) }
+  scope :unexpired, ->() { where('expiration > ?', Time.now) }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.